### PR TITLE
Re-add the layout and setup from #26

### PR DESCRIPTION
**Pull request by ethanwhite**
This Revert's "Revert "Updating layout and setup info"". To reinstate
the unreviewed layout changes.

This reverts commit 5d6d3f55a64abd4d4c83fe1ad937cc472dd06707.

### CODE REVIEW

**Comment by ethanwhite** [comment]
This reverts the revert in #27 (first time I've ever reverted a revert before, so that's cool), which should bring in all of @apawlik's changes from #26. This is basically just merging in the lesson template from https://github.com/datacarpentry/lesson-template.

It seems a little weird to me that we have the setup information for all the different tools included in every repo, but that seems like something to be address in the lesson-template repo rather that here. So, I'm +1 for merge since this is basically brining in line with the other repos. Since I'm technically submitting this PR someone else should press the big green button if they're happy with it. @hlapp? @apawlik?

File: index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+<!-- USING THIS LESSON TEMPLATE -->
+<!-- Lesson specific information is taken from the YAML header at the top of the page -->
+
+<!-- THE LESSON INFORMATION -->
+
+<!-- Get the information from _data/info.yml -->
+
+#Data Carpentry {{ page.topic }} for {{ page.domain }}
```
Comment (hlapp) [inline index.md]: I'd suggest we be consistent and put a space after the heading-level defining `#`s.

File: index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+**Content Contributors: {{page.contributors | join: ', ' %}}**
+
+
+**Lesson Maintainers: {{page.maintainers | join: ', ' %}}**
+
+<br> 
+
+
+####Lesson status: {{ page.status }}
```
Comment (hlapp) [inline index.md]: Same re: space after `#`s. Also, why jump from level 1 to level 4 heading?

File: index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## Data
+
+Data files for the workshop are available here: [{{page.dataurl %}}]({{page.dataurl %}})
+
+
+<br>
+
+<h2>Requirements</h2>
```
Comment (hlapp) [inline index.md]: Any reason this uses HTML tags rather than markdown for heading level?

File: _layouts/workshop.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  <body>
+    <div class="container">
+      <div class="span10 offset1">
+        {% include banner.html %}
+
+        <div class="hero-unit">
+          <div class="row-fluid">
+            <div class="span10">
+	      <h2>A Data Carpentry Workshop</h2>
+	      <h4>Organized by Stanford Libraries </h4>
```
Comment (hlapp) [inline _layouts/workshop.html]: Shouldn't the _Stanford Libraries_ be replaced with a placeholder (or a handlebar expression using YAML front matter or configuration variables?

**Comment by hlapp** [comment]
I made some comments, but I suspect given @ethanwhite's description these really need to be address upstream rather than here?

@ethanwhite there is still one `*~` file in there (`anySetup~`) - could you remove that? (Although I suspect that's also coming from upstream.)

If the issues I noted really are all upstream in the lesson template, would it be worth making the changes there first, and then updating the PR here? Perhaps at least create a PR for the template first because once merged the comments may be hard to retrieve.

**Comment by ethanwhite** [comment]
I just pushed changes to fix the header spacing, get rid of the reference to Stanford and remove the old backup file. I'll go ahead and make a related PR to upstream.

**Comment by hlapp** [comment]
@ethanwhite: sounds good. I'm merging this.

**Comment by tracykteal** [comment]
The setup instructions are an upstream issue. We did remove the _data directory in the upstream lesson-template. We can remove those after merge though.

**Comment by ethanwhite** [comment]
Submitted upstream in https://github.com/datacarpentry/lesson-template/pull/10

**Comment by apawlik** [comment]
Excellent! Thanks @ethanwhite @hlapp @tracykteal 

Indeed I noted some things as you pointed out. I wasn't quite sure where the template is heading so I haven't touched it too much. Will carry on with some work. Have an upcoming workshop at ELIXIR Slovenia in 2 weeks.

Will try to address some problems pointed out by https://github.com/datacarpentry/R-ecology/issues/14

**Pull request merged**